TS_src/client/src/components/Contacts.tsx

```diff
@@ -1,82 +1,95 @@
+import { CssBaseline, Container, Box, Typography, TextField, IconButton, Button, InputAdornment } from '@mui/material';
 import '@fontsource/roboto/300.css';
 import SearchIcon from '@mui/icons-material/Search';
 import axios from 'axios';
 import ContactList from './ContactList';
-// import { Routes, Route } from "react-router-dom";
-// import MyTripCard from "./MyTripCard";
-import { User, Trip } from './types';
-import Sidebar from './Sidebar';
-import Divider from '@mui/material/Divider';
-import { useNavigate } from 'react-router-dom';
+import { User, Trip } from "./types";
+import Sidebar from "./Sidebar";
+import Divider from "@mui/material/Divider";
+import { useNavigate } from "react-router-dom";
 
 interface Props {
 	userInfo: User;
-	contacts: Array<User>;
-	setContacts: Dispatch<SetStateAction<Array<User>>>;
+	contacts: any;
+	setContacts: Dispatch<SetStateAction<any>>;
 	setActiveComponent: Dispatch<SetStateAction<string>>;
 	logout: Function;
 	setUserTrip: Dispatch<SetStateAction<Trip>>;
-	userTrip: any;
 }
 
 const googleURL = process.env.GOOGLEMAPSAPIKEY;
 
-function Contacts({ userInfo, contacts, setContacts, setActiveComponent, logout, setUserTrip, userTrip }: Props) {
+function Contacts({ userInfo, contacts, setContacts, setActiveComponent, logout, setUserTrip} : Props) {
 	const navigate = useNavigate();
 
 	// hook to manage contacts checked from list
-
-	const [checkedContacts, setCheckedContacts] = useState<User[]>([]);
+	const [checkedContacts, setCheckedContacts] = useState<Array<User>>([]);
+	// hook to disable starting trip button if no contacts are checked
 	const [buttonDisabled, setButtonDisabled] = useState<boolean>(true);
-	const [submitted, clickSubmitted] = useState(false);
+	// hook to set status on whether trips has started, for navigation purposes
+	const [submitted, clickSubmitted] = useState<boolean>(false);
 
 	// Fetch GET request for contact and add to list:
 	const handleSubmit = async (event: any) => {
 		event.preventDefault();
+	
 		const phone_number = event.target[0].value.replaceAll(/[^0-9]/g, '');
 		event.target[0].value = '';
 
-		//fetch request to get contact info
 		try {
 			const response = await axios.get(`/api/users/${phone_number}`);
-
 			const contactData = response.data[0];
+			// if contact does not exist
 			if (!contactData.name) return;
 
-			// add user to array of contacts
-			const contactShown = contacts.reduce((acc, user) => {
+			// if user exists
+
+			// check if contact is already in user's contact list
+			const contactShown = contacts.reduce((acc: number, user: User) => {
+        console.log('inside reduce: ', contactData, userInfo)
 				if (user.phone_number === contactData.phone_number) ++acc;
 				return acc;
 			}, 0);
-			if (!contactShown) setContacts([...contacts, contactData]);
+
+			// if contact is not in user's contact list, add it to the list
+			if (!contactShown) {
+        setContacts([...contacts, contactData]);
```

Comment:
same here with odd spacing

Reply:
I attempted to correct the spacing. Hopefully, it's done.